apps/spruce/src/gql/queries/task-history.graphql

```diff
@@ -13,6 +13,16 @@ query TaskHistory($options: TaskHistoryOpts!) {
       execution
       order
       revision
+      tests(opts: { statuses: ["fail", "silentfail"] }) {
+        testResults {
+          id
+          logs {
+            urlParsley
+          }
+          status
+          testFile
+        }
+      }
```

Comment:
it seems like the tests aren't being fetched correctly for display tasks, if I'm testing correctly. I can't really tell why it's not working but it's important that it does since MongoDB almost exclusively uses display tasks on their waterfall

Reply:
Good catch. I fixed it here: https://github.com/evergreen-ci/evergreen/pull/8974